deps/src/main/java/com/microsoft/azure/sdk/iot/deps/transport/amqp/AmqpsConnection.java

```diff
@@ -5,8 +5,8 @@
 
 package com.microsoft.azure.sdk.iot.deps.transport.amqp;
 
+import com.microsoft.azure.proton.transport.ws.impl.WebSocketImpl;
```

Comment:
Just wondering, why did we move to this and who maintains this codebase?

Reply:
This Proton-J websocket implementation now only lives [here](https://github.com/Azure/qpid-proton-j-extensions) and we share ownership with the eventhubs SDK team. The motivation behind this was that both our teams needed a websocket implementation that works with proton, so it made sense for us to both use the same implementation.